salt/cache/redis_cache.py

```diff
@@ -37,73 +30,60 @@
     1) "my-key"
     127.0.0.1:6379> GET $KEY_root-bank/sub-bank/leaf-bank/my-key
     "my-value"
-
 There are three types of keys stored:
-
 - ``$BANK_*`` is a Redis SET containing the list of banks under the current bank
 - ``$BANKEYS_*`` is a Redis SET containing the list of keys under the current bank
 - ``$KEY_*`` keeps the value of the key
-
 These prefixes and the separator can be adjusted using the configuration options:
-
 bank_prefix: ``$BANK``
     The prefix used for the name of the Redis key storing the list of sub-banks.
-
 bank_keys_prefix: ``$BANKEYS``
     The prefix used for the name of the Redis keyt storing the list of keys under a certain bank.
-
 key_prefix: ``$KEY``
     The prefix of the Redis keys having the value of the keys to be cached under
     a certain bank.
-
 separator: ``_``
     The separator between the prefix and the key body.
-
 The connection details can be specified using:
-
 host: ``localhost``
     The hostname of the Redis server.
-
 port: ``6379``
     The Redis server port.
-
 cluster_mode: ``False``
     Whether cluster_mode is enabled or not
-
 cluster.startup_nodes:
     A list of host, port dictionaries pointing to cluster members. At least one is required
     but multiple nodes are better
-
     .. code-block::yaml
-
         cache.redis.cluster.startup_nodes
           - host: redis-member-1
             port: 6379
           - host: redis-member-2
             port: 6379
-
 cluster.skip_full_coverage_check: ``False``
     Some cluster providers restrict certain redis commands such as CONFIG for enhanced security.
     Set this option to true to skip checks that required advanced privileges.
-
     .. note::
-
         Most cloud hosted redis clusters will require this to be set to ``True``
-
 db: ``'0'``
     The database index.
-
     .. note::
-
         The database index must be specified as string not as integer value!
-
 password:
```

Comment:
None of the lines above should have been deleted, many are there for Sphinx formatting. Please put them back.